allauth/account/views.py

```diff
@@ -428,7 +428,7 @@ def form_valid(self, form):
         get_adapter(self.request).add_message(
             self.request,
             messages.INFO,
-            "account/messages/" "email_confirmation_sent.txt",
+            "account/messages/email_confirmation_sent.txt",
```

Comment:
use single cotation ( ' ) instead of duble ( " )

Reply:
@yasinByasin94 , in that package `black` is used.